lang/pt.js

```diff
@@ -7,25 +7,25 @@ export default {
 	"components.calendar.selected": "Selecionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Fechar esta caixa de diálogo",
-	"components.form-element.defaultError": "{label} é inválido.",
+	"components.form-element.defaultError": "Valor inválido para {label}.",
```

Comment:
Old: `{label} is invalid.`
New: `Invalid value for {label}.`

Looks like previous was correct